File: app/src/main/java/com/diploma/bank/TempData/UserAccounts.java
package com.diploma.bank.TempData;

public class UserAccounts {

    private String login;
    private String password;

    public static UserAccounts[] user_account = {
            new UserAccounts("admin", "admin"),
            new UserAccounts("user", "user"),
            new UserAccounts("", "")};

    private UserAccounts(String login, String password){
        this.login = login;
        this.password = password;
    }

    public String getLogin() {
        return login;
    }

    public String getPassword() {
        return password;
    }
}

File: app/src/main/java/com/vmbeshenov/bank/TempData/UserAccounts.kt
package com.vmbeshenov.bank.TempData

class UserAccounts private constructor(val login: String, val password: String) {

    companion object {
        var user_account = arrayOf(
                UserAccounts("admin", "admin"),
                UserAccounts("user", "user"),
                UserAccounts("", ""))
    }
}